src/main/java/pro/sky/telegrambot/entity/PetReport.java

```diff
@@ -0,0 +1,78 @@
+package pro.sky.telegrambot.entity;
+
+import com.pengrad.telegrambot.model.PhotoSize;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Table(name = "pet_report")
+public class PetReport {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(length = 20, nullable = false)
+    private String owner;
+
+    @Column(name = "date_time", nullable = false)
+    private LocalDateTime dateTime;
+
+    @Column(length = 20, nullable = true)
```

Comment:
```suggestion
    @Column(length = 20, nullable = true, columnDefinition = "oid")
```
чтобы не хранить байты в самой ячейке таблицы